recipes/deeppurpose/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "deeppurpose" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kexinhuang12345/DeepPurpose/archive/v{{ version }}.tar.gz
+  sha256: 3090375c9fca73ff24c4e0c2c82f0c3fca7c96e77eac66ba4ab5f84f28697a1c
+  
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools >=38.6.0
+  run:
+    - python >=3.6
+    - numpy
+    - pandas
+    - pytorch
+    - pandas-flavor
+    - lifelines
+    - prettytable
+    - scikit-learn
+    - tqdm
+    - rdkit >=2020.03.6
+    - requests
+
+test:
+  imports:
+    - DeepPurpose
+
+about:
+  home: https://github.com/kexinhuang12345/DeepPurpose
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'A Deep Learning Library for Compound and Protein Modeling DTI, Drug Property, PPI, DDI, Protein Function Prediction'
+  
+extra:
+  recipe-maintainers:
+    - kexinhuang12345
+    - futianfan
```

Comment:
@futianfan are you willing to be a maintainer of this recipe?

Reply:
Yes. Glad to be maintainer.